src/utils/py/daos_api.py

```diff
@@ -34,6 +34,15 @@
 from daos_cref import *
 from conversion import *
 
+# pylint: disable=no-member
+# pylint: disable=exec-used
+# pylint: disable=import-error
+DAOS_MAGIC = 0x7A89
+if sys.version_info < (3, 0):
+    from pydaos_shim_27 import *
+else:
+    from pydaos_shim_34 import *
```

Comment:
(pylint-wildcard-import) Wildcard import pydaos_shim_34

Reply:
To get around this maybe we should:
`import pydaos_shim_34 as pydaos_shim`